fhir-audit/src/main/java/com/ibm/fhir/audit/cadf/model/CadfResource.java

```diff
@@ -377,6 +377,9 @@ public static String generate(CadfResource obj)
         }
 
         public static void generate(CadfResource obj, JsonGenerator generator) throws IOException {
```

Comment:
maybe some javadoc to explain the behavior when null is passed?

Reply:
added comment